docs/hugo/content/guide/authentication/credential-format.md

```diff
@@ -779,5 +779,81 @@ spec:
 EOF
 ```
 
+## User Assigned Identity Credentials
+
+### General
+This option is only available for 1st party Microsoft applications who have access to the msi data-plane.
+
+This authentication type is similar to user assigned managed identity authentication combined with client certificate
+authentication. As a 1st party Microsoft application, one has access to pull a user assigned managed identity's backing
+certificate information from the MSI data plane. Using this data, a user can authenticate to Azure Cloud.
+
+### Prerequisites
+A JSON file with information from the user assigned managed identity. It should be in this format:
+```json
+        {
+            "client_id": "0998...",
+            "client_secret": "MIIKUA...",
+            "client_secret_url": "https://control...",
+            "tenant_id": "93b...",
+            "object_id": "ae...",
+            "resource_id": "/subscriptions/...",
+            "authentication_endpoint": "https://login.microsoftonline.com/",
+            "mtls_authentication_endpoint": "https://login.microsoftonline.com/",
+            "not_before": "2025-02-07T13:29:00Z",
+            "not_after": "2025-05-08T13:29:00Z",
+            "renew_after": "2025-03-25T13:29:00Z",
+            "cannot_renew_after": "2025-08-06T13:29:00Z"
+        }
+```
+
+Note, the client secret should be a base64 encoded certificate.
+
+The steps to get this information from the MSI data plane are as follows:
+1. Make an unauthenticated GET or POST (no Authorization request headers) on the x-ms-identity-url received from ARM to get the token authority and, on older api versions, resource.
+2. Get an Access Token from Azure AD using your Resource Provider applicationId and Certificate. The applicationId should match the one you added to your manifest. The response should give you an access token.
+3. Perform a GET or POST to MSI on the same URL from earlier to get the Credentials using this bearer token.
+
+The only required environment variable is AZURE_USER_ASSIGNED_IDENTITY_CREDENTIALS.
+```bash
+export AZURE_USER_ASSIGNED_IDENTITY_CREDENTIALS="/path/to/credentials-file.json" # The file path to the msi data plane credentials in a JSON file format.
+```
+
+It is expected this JSON file is available in a volume on the pod needing to authenticate with Azure cloud with this
+authentication method. For example, if the credentials were stored as a secret in Azure Key Vault, one could use the 
+Secrets CSI Driver and a SecretProviderClass custom resource to automatically have the file mounted into a volume on a 
+pod.
+
```

Comment:
```suggestion

{{% alert title="Warning" color="warning" %}}
No option is exposed to configure additional deployment volumes in the Helm chart because it is reserved for use by Microsoft 1st party access and not intended for 3rd party use.
{{% /alert %}}
```

(Or similar, I'm not too picky on the wording)